### ISSUE TITLE
When moving character by character in the address bar of the hedge the nvda does not read correctly

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#8817]
### Steps to reproduce:
Open Microsoft hedge.
In the address bar enter any address.
Now press home to go to the beginning of the text and walk with arrows to the right.
### Actual behavior:
Note that nvda does not read the characters correctly, some times the nvda reads the same character twice, skips the reading of other characters when it repeats the previous one.
### Expected behavior:
the nvda should read the characters according to the focus reached with the right and left arrows.
#### NVDA Installed/portable/running from source:
intsal
#### NVDA version:
alfa 16123
#### Windows version:
10 17763.1
#### Name and version of other software in use when reproducing the issue:
microsoft hedge
#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA?
no
[hedge.txt](https://github.com/nvaccess/nvda/files/2443191/hedge.txt)

**Comment by Brian1Gaff** [comment]
Is this one of those pesky autocomplete fields or one that has a vertical 
list of suggestions making it about as speedy as treacle in a freezer?
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
-----

**Comment by leonardder** [comment]
I can't reproduce this with alpha-16149 and Windows 10 1809 (17763.1)

**Comment by fernando-jose-silva** [comment]
I was able to find the problem in another machine executing the windows 10 17763.1 using the nvda 16149 alpha and another synthesizer in the case now vocalizer, in the previous computer you are one core.
follows another log:
In addition to this I opened the hedge pressi ctrl + l, typed www.google.com
press home and then right arrow and I noticed that nvda repeats some letters and skips others, this is more perspitable when the address is long, since nvda does not do this in all characters.
[google1.txt](https://github.com/nvaccess/nvda/files/2453564/google1.txt)

**Comment by Adriani90** [comment]
This works as expected in Edge Chromium 81 with NVDA 2020.1. Edge Chromium will be distributed to most users via Windows 10 2004 Update, so I am closing this as works for me while adding the propper label "Edge Spartan" since it might still be reproducible in the old Edge.

**Comment by josephsl** [comment]
Hi, two things: Edge 81 will not come as part of Version 2004 unless Microsoft says otherwise, and this issue might not be present in classic Edge due to editable text enhancements. Thanks.

**Comment by Adriani90** [comment]
Hi Joseph,

 

I think this is not entirely correct. according to this article the Edge Chromium will be distributed:

https://www.windowslatest.com/2020/05/09/new-microsoft-edge-is-rolling-out-to-windows-10-may-2020-update/

 

 

Von: Joseph Lee <notifications@github.com> 
Gesendet: Sonntag, 10. Mai 2020 02:55
An: nvaccess/nvda <nvda@noreply.github.com>
Cc: Adriani90 <adriani.botez@googlemail.com>; State change <state_change@noreply.github.com>
Betreff: Re: [nvaccess/nvda] When moving character by character in the address bar of the hedge the nvda does not read correctly (#8817)

 

Hi, two things: Edge 81 will not come as part of Version 2004 unless Microsoft says otherwise, and this issue might not be present in classic Edge due to editable text enhancements. Thanks.

**Comment by josephsl** [comment]
Hi, that’s for automatic rollout. We’ll see if that claim is true by end of May at the earliest. Thanks.
